extensions/amp-story/1.0/test/test-amp-story-page.js

```diff
@@ -410,6 +413,7 @@ describes.realWin('amp-story-page', {amp: {extensions}}, (env) => {
   });
 
   it('should use storyNextUp value as default for auto-advance-after', async () => {
+    initializePageWithElement;
```

Comment:
Nit: Did you mean to call this function?

Reply:
Fixed, thanks for catching that!